openedx/core/lib/xblock_utils.py

```diff
@@ -240,7 +240,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         else:
             return frag
 
-    if isinstance(block, SequenceModule):
+    if isinstance(block, SequenceModule) or getattr(block, 'HIDDEN', False):
```

Comment:
@cpennington There should be no reason that instructors should need to look at student state. It also wouldn't be clear if we left the staff debug info, it wouldn't be clear what component that relates to. I would leave staff debug out but let me know if I'm missing something.